core/lib/spree/core/controller_helpers/order.rb

```diff
@@ -21,10 +21,10 @@ def simple_current_order
           @simple_current_order = find_order_by_token_or_user
 
           if @simple_current_order
-            @simple_current_order.last_ip_address = ip_address
+            @simple_current_order.update(last_ip_address: ip_address)
```

Comment:
I agree this should be an `update!`. Also before merge, there is this question about my fixup commit in this PR: https://github.com/solidusio/solidus/pull/1658#issuecomment-269883417